store/getters.js

```diff
@@ -623,6 +624,14 @@ function existsOnce(name, list) {
   }
 }
 
+function hasMoreTumours(list) {
+  let tumourAmount = 0;
+  list.forEach((sample) => {
+    if (sample.tumour) tumourAmount++;
+  });
+  return tumourAmount > 1 ? true : false;
```

Comment:
```suggestion
  let tumourCount = 0;
  list.forEach((sample) => {
    if (sample.tumour) tumourCount++;
  });
  return tumourCount > 1 ? true : false;
```
Amount sound like a volume or weight, right?